src/lib.rs

```diff
@@ -54,6 +59,7 @@ pub struct RocketSentry {
 #[derive(Deserialize)]
 struct Config {
     sentry_dsn: String,
+    sentry_transaction_sample_rate: Option<f32>,  // Default is 0 so no transaction transmitted
```

Comment:
Let's keep configuration option names in sync with Sentry's.

```suggestion
    sentry_traces_sample_rate: Option<f32>,  // Default is 0 so no transaction transmitted
```